public/views/index.pug

```diff
@@ -0,0 +1,155 @@
+                                                                span.input-group-append
+                                                                    button.btn.btn-outline-secondary.border-left-0.border(type='button' data-toggle='popover' data-trigger='focus' title='Wallet Address' data-content='Enter your wallet address so you can receive your reward.')
+                                                                        i.fas.fa-info
+                                                        .form-group
+                                                            .input-group
+                                                                span.input-group-prepend
+                                                                    span.input-group-text.bg-transparent.border-right-0
+                                                                        i.fab.fa-twitter.fa-2x
+                                                                input#tweetUrl.form-control.form-control-lg.py-2.border-left-0.border(name='tweetUrl' type='text' placeholder='Enter tweet url...')
+                                                                span.input-group-append
+                                                                    button.btn.btn-outline-secondary.border-left-0.border(type='button' data-toggle='popover' data-trigger='focus' title='Tweet Url' data-content='First, you need to tweet about Lisinski Faucet PIPA and Lisinski Ethereum Testnet, then paste tweet URL here.')
+                                                                        i.fas.fa-info
+                                                        .form-group
+                                                            a.twitter-share-button(href=predefinedTweetUrl data-size='large')
+                                                                | Tweet about Lisinski
+                                                        .form-group
+                                                            .g-recaptcha(data-sitekey=sitekey style='transform:scale(1);-webkit-transform:scale(1);transform-origin:0 0;-webkit-transform-origin:0 0;')
+                                                        .form-group
+                                                            button#requestTokens.request-tokens-button(type='submit') Request #{maxAmount} LisinskiEther (LETH)
+                                                .card-footer.text-muted
+                                                    .footer-note
+                                                        | After you claimed your #{maxAmount} LETH award, you can claim new award after #{cooldown} hours.
+                                                    .footer-button
+                                                        button.request-tokens-button.gray-button.toggle I want #{minAmount} LETH
```

Comment:
Not sure, but I think you need to format those amounts to ether because they are in miliether